### PR TITLE
Issue#17 - Add Progress Bar to show progress on perturbations and scores

### PR DESCRIPTION
**Pull request by jimilp7**
I would like to contribute!

Solves [Issue#17](https://github.com/fiddler-labs/fiddler-auditor/issues/17)

Requires: [TQDM](https://tqdm.github.io/): pip install tqdm

Sample Run:
Screenshot 2023-09-29 at 4 46 56 PM

Comments: These are the places I could find computations for perturbations, if I am missing more, please let me know and I can add those in.

### CODE REVIEW

**Comment by iterix** [comment]
@jimilp7 Thank you for the contribution! 💥

File: auditor/evaluation/discriminative.py [hunk 1]
```diff
@@ -90,6 +91,8 @@ def evaluate(
             f'Started model evaluation with perturbation type '
             f'{self.perturbed_dataset.perturbation_type}'
         )
+        progress_bar = ProgressLogger(total_steps=min(len(self.perturbed_dataset.data), len(self.perturbed_dataset.metadata)), description="Starting Model Evaluation")
```
Comment (iterix) [inline auditor/evaluation/discriminative.py:94]: To get past lint check can you pass the arguments in the next line.

`./auditor/evaluation/discriminative.py:94:81: E501 line too long (167 > 80 characters)`

**Comment by iterix** [comment]
@jimilp7 Loved the addition of progress logger under utils.

Suggested minor changes to comply with lint check. We are good to merge after that. To run lint check locally just use

`pip install flake8`
`make lint`

for detailed environment set-up instructions you can refer to https://github.com/fiddler-labs/fiddler-auditor/blob/main/CONTRIBUTION.md

**Comment by jimilp7** [comment]
> @jimilp7 Loved the addition of progress logger under utils.
> 
> Suggested minor changes to comply with lint check. We are good to merge after that. To run lint check locally just use
> 
> `pip install flake8` `make lint`
> 
> for detailed environment set-up instructions you can refer to https://github.com/fiddler-labs/fiddler-auditor/blob/main/CONTRIBUTION.md

@iterix Thank you so much for looking into this, just fixed the linter problem!

**Comment by iterix** [comment]
@jimilp7 Thanks for the quick turnaround! Approved 🚀

**Pull request merged**